### PR TITLE
Feature/gallery-page-template

### PR DESCRIPTION
**Pull request by Atanas-F-Nikolov**
- Basic template of the "component library"
- header
- menu
- gallery container

### Screenshots 
image

### CODE REVIEW

File: src/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+// import { NavLink } from 'react-router-dom';
```
Comment (kblomster) [inline src/Header.tsx]: I don't think we should commit commented out code

File: src/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+export default class Header extends React.Component {
+
```
Comment (kblomster) [inline src/Header.tsx]: Do you like an empty line between the opening { and the first line of code in a class?

File: src/Menu.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { NavLink } from 'react-router-dom';
+export default class Menu extends React.Component {
```
Comment (kblomster) [inline src/Menu.tsx]: I like an empty line between the last `import` and the first line of code in the file. What do you think?

File: src/App.tsx [hunk 4]
```diff
@@ -1,20 +1,33 @@
...
+                <Route component={NotFoundPage} />
+              </Switch>
+            </div>
+          </div>
         </div>
-        <p className="App-intro">
-          To get started, edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <Button size="small" text="Hejsan" />
+        {/* <Footer /> */}
```
Comment (kblomster) [inline src/App.tsx]: Commented out code, remove?

File: src/Menu.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+              activeClassName="active"
+              className="menu__list-item--link">
+              Knappar
+              </NavLink>
+          </li>
+        </ul>
+      </div>
+    );
+  }
+}
```
Comment (kblomster) [inline src/Menu.tsx]: Add blank line at end of file

File: src/Header.tsx [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+  render() {
+    return (
+      <div className="header">
+        <div className="header__container">
+          <div className="header__container-logo" />
+        </div>
+      </div>
+    );
+  }
+}
```
Comment (kblomster) [inline src/Header.tsx]: Add new line at end of file

File: src/assets/styles/app.scss [hunk 7]
```diff
@@ -1,5 +1,52 @@
...
+@import
+   'objects/grid',
+   'objects/space';
+
+// 6. Trumps
+@import 
+    'trumps/index';
+
+// 7. Components
+@import 'components/button/button';
```
Comment (kblomster) [inline src/assets/styles/app.scss]: Style imports the same as the ones above?

File: src/assets/styles/elements/_figure.scss [hunk 8]
```diff
@@ -0,0 +1,10 @@
+///
+/// Figures
+/// <figure>
+///============================================================================
+
+figure {
+  // Normalize adds `margin` to `figure`s as browsers apply it inconsistently.
+  // We reset that to create a better flow in-page.
+  margin: 0 0 1rem;
+}
```
Comment (kblomster) [inline src/assets/styles/elements/_figure.scss]: New line at end of file

File: src/assets/styles/elements/_button.scss [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+  // Work around a Firefox/IE bug where the transparent `button` background
+  // results in a loss of the default `button` focus styles.
+  //
+  // Credit: https://github.com/suitcss/base/
+  &:focus {
+    outline: 1px dotted;
+    outline: 5px auto -webkit-focus-ring-color;
+  }
+}
```
Comment (kblomster) [inline src/assets/styles/elements/_button.scss]: New line at end of file

File: src/components/button/Button.tsx [hunk 10]
```diff
@@ -2,13 +2,13 @@ import * as React from 'react';
...
   text: string;
-  size: 'small' | 'medium' | 'large';
+  size: 'small' | 'standard' | 'large';
 }
 
 export default class Button extends React.Component<ButtonParameters, object> {
   render () {
     return (
-      <button className={'btn ' + this.props.size}>
+      <button className={`btn btn--${this.props.size}`}>
```
Comment (kblomster) [inline src/components/button/Button.tsx]: Nice!

File: src/gallery-pages/AlertPage.tsx [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+export default class AlertPage extends React.Component {
+
+  render() {
+    return (
+      <div>
+        <h1>Alert</h1>
+      </div>
+    );
+  }
+}
```
Comment (kblomster) [inline src/gallery-pages/AlertPage.tsx]: New line at end of file

File: src/gallery-pages/ButtonPage.tsx [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+          <Button size="standard" text="Hejsan" />
+        </div>
+        <div className="spacer-mb-medium">
+          <h4>Stor knapp</h4>
+          <Button size="large" text="Hejsan" />
+        </div>
+      </div>
+    );
+  }
+}
```
Comment (kblomster) [inline src/gallery-pages/ButtonPage.tsx]: New line at end of file

File: src/gallery-pages/ButtonPage.tsx [hunk 13]
```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import Button from '../components/button/Button';
+export default class ButtonPage extends React.Component {
+
```
Comment (kblomster) [inline src/gallery-pages/ButtonPage.tsx]: No whitespace between opening brace and first line of code in the block

File: src/gallery-pages/ButtonPage.tsx [hunk 14]
```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import Button from '../components/button/Button';
+export default class ButtonPage extends React.Component {
```
Comment (kblomster) [inline src/gallery-pages/ButtonPage.tsx]: Add a blank line between last import statement and first line of code

File: src/gallery-pages/WelcomePage.tsx [hunk 15]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+export default class WelcomePage extends React.Component {
+
```
Comment (kblomster) [inline src/gallery-pages/WelcomePage.tsx]: No blank line here

File: src/gallery-pages/WelcomePage.tsx [hunk 16]
```diff
@@ -0,0 +1,12 @@
...
+  render() {
+    return (
+      <div>
+        <h3>Välkommen till komponent biblioteket!</h3>
```
Comment (kblomster) [inline src/gallery-pages/WelcomePage.tsx]: "komponentbiblioteket" är ett ord :)

File: src/gallery-pages/WelcomePage.tsx [hunk 17]
```diff
@@ -0,0 +1,12 @@
...
+      </div>
+    );
+  }
+}
```
Comment (kblomster) [inline src/gallery-pages/WelcomePage.tsx]: New line at end of file

File: src/App.tsx (hunk 4 above)
Comment (Atanas-F-Nikolov) [reply src/App.tsx]: removed

File: src/Header.tsx (hunk 1 above)
Comment (Atanas-F-Nikolov) [reply src/Header.tsx]: no

**Pull request merged**